python/tests/test_variable.py

```diff
@@ -589,7 +590,11 @@ def test_construct_0d_numpy():
 
 
 def test_construct_0d_native_python_types():
-    assert sc.Variable(2).dtype == sc.dtype.int64
+    if sys.platform == "win32":
+        # Windows defaults to a 32 bit integer
+        assert sc.Variable(2).dtype == sc.dtype.int32
+    else:
+        assert sc.Variable(2).dtype == sc.dtype.int64
```

Comment:
I don't understand, this does not make sense. Why?

Reply:
As discussed above.  `long` 4 bytes windows, 8 bytes linux.

Reply:
😱

Reply:
(I mean, I knew that this is what C/C++ do, but I had assumed Python would not follow this track)